subprojects/packagefiles/luajit/meson.build

```diff
@@ -1,18 +1,47 @@
 project('luajit', 'c', version : '2.0.5', license : 'mit')
 
 # TODO: This is not ideal. We should translate the enough of the makefile to have Meson handle the build.
+cc = meson.get_compiler('c')
+
+env = ['DESTDIR="' + (meson.current_source_dir() / 'out') + '"']
```

Comment:
Since this isn't inline anymore, it might be good if it was an environment object instead of an array.

Reply:
Thanks, didn't know about those. Will look into it after work.